cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ClusterCa.java

```diff
@@ -115,7 +116,7 @@ public Secret cruiseControlSecret() {
         return cruiseControlSecret;
     }
 
-    public Map<String, CertAndKey> generateZkCerts(Kafka kafka, boolean isMaintenanceTimeWindowsSatisfied) throws IOException {
+    public Map<String, CertAndKey> generateZkCerts(Reconciliation reconciliation, Kafka kafka, boolean isMaintenanceTimeWindowsSatisfied) throws IOException {
```

Comment:
Do we need the param here, given it's non-static and we passed the Reconciliation in the constructor?